### ISSUE TITLE
NVDA does not announce selection for mathematical content

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5104]
**Reported by bhavyashah on 2015-05-23 10:18**
I am using NVDA's Math try build, Math Player 4 Beta2 and Microsoft Office Word 2007.
If I try selecting mathematical content using standard Windows selection commands including Shift + left and right arrows, mathematical content does get selected, because I am able to copy it and get the selected result pasted, however, NVDA does not announce selection of mathematical content appropriately or like it correctly does for normal text.
Simply try to select mathematical content by pressing Shift + right arrow for instance, and, notice that NVDA simply says selected, and does not continue to say what has been selected, i.e. the mathematical content.
**Blocked by #4877**

**Comment by Adriani90** [comment]
This works as expected now in MS Office 365 version 2505 with NVDA 2025.1.2 RC. Closing as works for me.
Please comment if you are still having this issue and we can reopen.
